team-telnyx/webrtc
# docs(WEBRTC-231): add typedoc on newCall method

- add typedoc on newCall method

## 📝 To Do


## ✋ Manual testing

1. Navigate to `packages/js/example/react-audio` and run `npm start`.
2. Access http://localhost:6006/?path=/story/clicktocall--example and provide your SIP credentials.
3. Open a softphone and connect there you can you [this webdialer](https://webrtc.telnyx.com/rtc/index.html).
4. Go to Web Dialer and provide another SIP credentials on Web dialer and click in connect. 
5. Provide your `destinationNumber`.
6. Come back to http://localhost:6006/?path=/story/clicktocall--example and click on the button `Call`. 
7. You should go to Web dialer and pick up the call.

## 🦊 Browser testing

### Desktop


![image](https://user-images.githubusercontent.com/16343871/101167209-51894e00-3618-11eb-870a-fc88d72aab62.png)
